docs/concepts/add-in-development-best-practices.md

```diff
@@ -136,4 +136,4 @@ See [Support older Microsoft webviews and Office versions](../develop/support-ie
 ## See also
 
 - [Office Add-ins platform overview](../overview/office-add-ins.md)
-- [Learn about the Microsoft 365 Developer Program](https://developer.microsoft.com/microsoft-365/dev-program)
+- [Learn about the Microsoft 365 Developer Program](/office/developer-program/microsoft-365-developer-program-faq#who-qualifies-for-a-microsoft-365-e5-developer-subscription-)
```

Comment:
This originally linked to the Dev Program landing page; not sure that you want to link to the FAQ in this case.